pym/bob/scm/url.py

```diff
@@ -98,24 +98,45 @@ def asScript(self):
 cp -uT {URL} {FILE}
 """
         else:
+            tool = "curl"
+            options = ""
+            useTimeStamping = True
+            if which(tool) is not None:
+                options += "-sSgLf"
+                if not self.__sslVerify: options += "k"
+                options += " -o"
+            elif which("wget") is not None:
+                protocol = self.__url.split(":", 1)[0]
+                if protocol in ["http", "https", "ftp"]:
+                    Info("'curl' (preferred) wasn't found and 'wget' used as fallback instead.").show()
+                    tool = "wget"
+                    options += "-q --no-glob"
+                    if not self.__sslVerify: options += "--no-check-certificate"
+                    options += " -O"
+                    # Since time-stamping can't be used with the "-O" option together
+                    useTimeStamping = False
+                else:
+                    raise ParseError("Fallback to 'wget' impossible due to protocol limitations. Please install 'curl'!")
+            else:
+                raise ParseError("Don't know how to download. Please install 'curl'!")
             tpl = """
 {HEADER}
 mkdir -p {DIR}
 cd {DIR}
-if [ -e {FILE} ] ; then
-    curl {OPTIONS} -o {FILE} -z {FILE} {URL}
+if [ -e {FILE} -a {USETIMESTAMPING} = "True" ] ; then
```

Comment:
Now the code will always download the file if wget is used. This will typically be unncessary in case the checkout step is not determinstic as a whole. Instead I would suggest to download only if the destination does not exist in case of wget.

Reply:
You're right, will be fixed as well.